scripts/parse_live.py

```diff
@@ -89,12 +90,37 @@ def query_pageids(wiki, pageids):
             text = d(text)
             yield (id, title, text)
 
+def query_revids(revids):
```

Comment:
I suspect you don't actually need to add a separate method here. `query_pageids` is already doing almost the same query -- it just doesn't pass the `revids` API parameter. You could just add a new argument to the existing `query_pageids`, no?

Reply:
Yeah they do almost the same query. I didn't look into yamwapi before :neutral_face:, so I’ll remove `query_revids` and modify `query_pageids`.